assets/js/state/selectors/activityLogsSettings.js

```diff
@@ -0,0 +1,36 @@
+import { createSelector } from '@reduxjs/toolkit';
+
+export const getActivityLogsSettings = createSelector(
+  [({ activityLogsSettings }) => activityLogsSettings],
+  ({
+    settings,
+    errors,
+    loading,
+    editing,
+    networkError,
+    testingConnection,
```

Comment:
`testingConnection` does not need to be here.
```suggestion
```

Reply:
done in https://github.com/trento-project/web/pull/2727/commits/918148f8bf5d14916869a51e02ccdd8ba017f637

Reply:
Removed in https://github.com/trento-project/web/pull/2727/commits/e2f68277aea2579e811502f8bde632dec4692e82